style.css

```diff
@@ -223,8 +236,8 @@ body {
 }
 
 button {
```

Comment:
- [ ] As mentioned by the previous Reviewer, Please ensure that your button matches the given design by reducing the `border-radius: 16px;` from `16px` to `8px` as indicated in the figma design. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

 
![Student Button](https://github.com/denigit5/Portfolio/assets/43528594/6aec7c15-793b-41a0-b3da-c09e462d1748)
 
</td>
      <td>

![Figma Button](https://github.com/denigit5/Portfolio/assets/43528594/0cff0b6c-29d8-4321-a87b-279820ff9945)

</td>
    </tr>
  </table>